operator/v1/types_ingress.go

```diff
@@ -1684,3 +1716,69 @@ type IngressControllerList struct {
 	metav1.ListMeta `json:"metadata,omitempty"`
 	Items           []IngressController `json:"items"`
 }
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:subresource:status
+// +kubebuilder:object:root=true
+//
+// IngressOperator contains configuration options specific to the Ingress Operator itself
+//
+// Compatibility level 1: Stable within a major release for a minimum of 12 months or 3 minor releases (whichever is longer).
+// +openshift:compatibility-gen:level=1
+type IngressOperator struct {
+	metav1.TypeMeta `json:",inline"`
+	metav1.ListMeta `json:"metadata,omitempty"`
```

Comment:
This should be objectmeta, this is what's preventing the generator from running correctly
```suggestion
	metav1.ObjectMeta `json:"metadata,omitempty"`
```